### PR TITLE
Fixes `pint` command

### PR DESCRIPTION
**Pull request by nunomaduro**
Before, simply having `pint` will try to run the global installation of Laravel Pint, which most users don't have. However, because `pint` comes by default on any Laravel installation - it's safe to use `./vendor/bin/pint`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ class SoloServiceProvider extends ServiceProvider
             ->addLazyCommands([
                 'Queue' => 'php artisan queue:listen --tries=1',
                 // 'Reverb' => 'php artisan reverb:start',
-                // 'Pint' => 'pint --ansi',
+                // 'Pint' => '/vendor/bin/pint --ansi',
```
Comment (gayansanjeewa) [inline README.md:95]:
```suggestion
                // 'Pint' => './vendor/bin/pint --ansi',
```
Missing `.` for the current directory - typo I believe :)

**Comment by aarondfrancis** [comment]
Thanks for the PR! This is a little tricky to me though... I have Pint installed globally and not locally. I'm sure some people are the exact opposite. Changing it either way will provoke PRs in the opposite direction! I think since this is a stub file we'll just leave it as is unless someone has a good argument.

**Comment by aarondfrancis** [comment]
Ok I immediately changed my mind. If it's included with Laravel then that's good enough for me.

**Comment by joshmanders** [comment]
@aarondfrancis this is where my suggestion previously of putting `./vendor/bin` and potentially `./node_modules/.bin` into the `$PATH` used in the process would resolve. If the user has the command in either node dependencies, for ex `tailwindcss` binary, or in their `vendor/bin` such as with `pint` it resolve those, otherwise it will traverse up the `$PATH` until it hits your globally installed version.

I can submit a PR for this if you'd like.

**Pull request merged**